dts/arm/nxp/nxp_rt11xx.dtsi

```diff
@@ -1230,3 +1230,7 @@
 	 */
 	status = "disabled";
 };
+
+zephyr_cam_i2c: &lpi2c6 {};
```

Comment:
This seems board specific and probably belongs in the board overlay file.

Reply:
yes, it is pointed out by Daniel and I addressed it in the split PR.